tutorials/installation.md

```diff
@@ -29,6 +29,10 @@ $ cd napari
 $ pip install -e .
 ```
 
+### Linux dependencies
+
+On Linux, additional dependencies may be needed.  On Arch Linux, install [qt5-tools](https://www.archlinux.org/packages/extra/x86_64/qt5-tools/) may be needed to enable napari from an ipython session
```

Comment:
To late now but I think the grammar of the last sentence is weird. Perhaps "On Arch Linux, you may need to install [qt5-tools](https://www.archlinux.org/packages/extra/x86_64/qt5-tools/) to enable napari from an ipython session"?

Reply:
Never too late for another PR! ;) But let's wait until we can find exactly what the issue is, since you couldn't reproduce...

Reply:
Woops. sorry about the grammar problem.  Happy to fix when ready.